### DISCUSSION TITLE
lastversion cannot find GitHub api token in env variables

### DISCUSSION THREAD

**Discussion by FinSmith198** [dvershinin/lastversion#143] [Q&A] [answered]
to make sure the environmental variable is present
![image](https://github.com/dvershinin/lastversion/assets/137229198/167915c5-2f79-42b3-8888-a5a58f1471fd)

but attempting to pull from a private repo, i get that the api key was not found
![image](https://github.com/dvershinin/lastversion/assets/137229198/053317f1-fb4b-4264-8d5f-bafd03060fcd)

am i doing something wrong?

**Comment by dvershinin** [comment, marked as answer]
Can't really tell. Perhaps you run `env` in another tab where environment variable is not set, virtual environment specifics., etc. `lastversion` uses `os.getenv` it's like any other program out there...  but if you're absolutely sure it's a bug let me know

**Reply by FinSmith198** [reply]
Yes, it was because i am not used to linux, and i mistakenly added 'sudo' to the lastversion command, which ran it as root user, and not on my current user, so it didn't see the environment variable, which was not on root user. it seems to be fixed now, thanks for the help
